README.md

```diff
@@ -1,2 +1,67 @@
-# reviews-php
-A review service for PHP applications.
+
+
+#  Reviews-php
+[![Build Status](https://travis-ci.com/Pod-Point/reviews-php.svg?branch=master)](https://travis-ci.com/Pod-Point/reviews-php) [![codecov](https://codecov.io/gh/Pod-Point/reviews-php/branch/master/graph/badge.svg)](https://codecov.io/gh/Pod-Point/reviews-php)
+
+A review package for our PHP applications. Currently only includes reviews.co.uk but there are plans to add Trustpilot.
+
+##  Installation
+
+Follow the steps below in the desired project where you would like to install this package
+
+ 1. Add the following in the repository section of the composer.json .
+```json
+{  
+    "type": "git",  
+    "url": "git@github.com:pod-point/reviews-php.git"  
+}
+```
+2. Then add to this line providers section in the app.php config file.
```

Comment:
since we only support laravel > 5.5 with package discovery we don't need this step